Modules/Remote/RTK.remote.cmake

```diff
@@ -2,5 +2,5 @@
 itk_fetch_module(RTK
     "Reconstruction Toolkit (RTK) http://www.openrtk.org/"
   GIT_REPOSITORY ${git_protocol}://github.com/SimonRit/RTK.git
-  GIT_TAG v2.0.1
+  GIT_TAG 99a6626f42e6654d11a780127610c95d3c1a063e
```

Comment:
`RTK` is one of the few modules, if not the only one, that is tagged consistentl and uses these instead of commit hashes to indicate the appropriate version to be used within ITK.

This is a case that is not currently handled properly by the update script.

Reply:
I will undo this change, then.

Reply:
I admit I'm not following this conversation. Is that fine if I tag RTK before an ITK release after checking that everything compiles ok? How do you handle patch release for remote modules, can I have a non-patch release for RTK included in a patch release of ITK?

Reply:
Remote modules are far less strictly controlled than main ITK repository. Most remote modules don't have tags and versions, and we just take the latest commit in master branch in most cases.